config/prow/cluster/renovate/helm/values.yaml

```diff
@@ -10,7 +10,18 @@ renovate:
     {
       "platform": "github",
       "autodiscover": true,
-      "autodiscoverFilter": ["gardener/*"],
+      "autodiscoverFilter": [
+        "gardener/ci-infra",
+        "gardener/gardener",
+        "gardener/gardener-extension-registry-cache",
+        "gardener/gardener-extension-shoot-oidc-service",
+        "gardener/dependency-watchdog",
+        "gardener/etcd-druid",
+        "gardener/gardener-extension-networking-cilium",
+        "gardener/gardener-extension-networking-calico",
+        "gardener/gardener-extension-shoot-rsyslog-relp",
+        "gardener/landscaper"
+      ],
```

Comment:
With this config, renovate will open onboarding PRs on all listed repositories. Is this what you want?
I see some of them use dependabot.

Reply:
That's ok, it is just ne PR. This way, they know about the 🤖 and can take it or leave it.